scripts/run_cppcheck.sh

```diff
@@ -45,6 +45,7 @@ if [ -z "$CPPCHECK_FLAGS" ]; then
     CPPCHECK_FLAGS="--quiet --force --std=c11 --std=c++11 --inline-suppr \
                     --enable=warning --error-exitcode=1 -i third_party \
                     --suppress=uninitMemberVar:ulalaca/ulalaca.cpp \
+                    --suppress=shiftTooManyBits:libxrdp/xrdp_mppc_enc.c \
```

Comment:
Personally I'd prefer to add this inline to the code - see (e.g.) `sesman/chansrv/chansrv_fuse.c:2779` for an example. Or is there a good reason why it's better here?

Reply:
No real reason. Possibly because it is several warnings and I suppressed the other ones like this.